tidb-lightning/monitor-tidb-lightning.md

```diff
@@ -23,24 +23,14 @@ pprof-port = 8289
 ...
 ```
 
-and in `tikv-importer.toml`:
-
-```toml
-# Listening address of the status server.
-status-server-address = '0.0.0.0:8286'
-```
-
```

Comment:
Should not be deleted because it's the newest update to the website: https://docs.pingcap.com/zh/tidb/dev/monitor-tidb-lightning
https://github.com/pingcap/docs-cn/pull/4094/files#diff-8f2d0e1534121abfce97fb00aadf49eeR27-R32

Reply:
Thanks! Add back in https://github.com/pingcap/docs/pull/3660/commits/ff53d99da94806f17563b5059d03fcdee83e3fd5.